package/boot/uboot-sunxi/Makefile

```diff
@@ -349,7 +359,7 @@ UBOOT_TARGETS := \
 UBOOT_CONFIGURE_VARS += USE_PRIVATE_LIBGCC=yes
 
 UBOOT_MAKE_FLAGS += \
-	BL31=$(STAGING_DIR_IMAGE)/bl31.bin
+	BL31=$(STAGING_DIR_IMAGE)/bl31_$(ATF).bin
```

Comment:
Looks like we could just use UENV? Or would that not be tidy enough?

Reply:
Yeah, thought about that, but I'd rather leave the two separate - there could always be a board where a separate UENV file have to be used, there were examples earlier for that.

Reply:
Okay, totally fine with that, we do similar stuff for build variables for devices in a few cases ...

Reply:
Thanks. I'll cleanup the rest and push a new one.